tests/dummy/app/router.js

```diff
@@ -1,12 +1,14 @@
 import Ember from 'ember';
 import config from './config/environment';
 
-const Router = Ember.Router.extend({
+const { Router } = Ember;
+
+const EmRouter = Router.extend({
```

Comment:
@kellyselden But the official blueprint has a global sweetening, why not follow the Dockyard styleguide where we reduce global calls?

https://github.com/DockYard/styleguides/blob/master/engineering/ember.md#import-what-you-use-do-not-use-globals